api-operator/pkg/apim/importAPI.go

```diff
@@ -139,6 +223,133 @@ func importAPIFromZip(config *corev1.ConfigMap, token string, endpoint string) e
 	return nil
 }
 
+// handleDeploymentValues Handle Param Values and Cert Values
+func handleDeploymentValues(zippedData []byte, paramsCM *corev1.ConfigMap, certsCM *corev1.ConfigMap) (string, error) {
+
+	swaggerDirectory, _ := ioutil.TempDir("", "api-swagger-dir*")
+
+	deploymentDir := filepath.Join(swaggerDirectory, filepath.FromSlash(Deployment))
+	err := utils.CreateDirIfNotExist(deploymentDir)
+	if err != nil {
+		return "", err
+	}
+
+	if paramsCM.Name != "" {
+
+		paramsFileName, err := maps.OneKey(paramsCM.Data)
+		if err != nil {
+			logImport.Error(err, "Error in the params configmap ", "params.yaml", paramsCM.Data)
+			return "", err
+		}
+		paramsData := paramsCM.Data[paramsFileName]
+
+		paramsContent, err := getParamValues(paramsData)
+		if err != nil {
+			logImport.Error(err, "Error in getting the params values ", "param data", paramsData)
+			return "", err
+		}
+
+		paramsFile := filepath.Join(swaggerDirectory, filepath.FromSlash(Deployment+"/intermediate_params.yaml"))
+		err = ioutil.WriteFile(paramsFile, paramsContent, os.ModePerm)
+		if err != nil {
+			logImport.Error(err, "Error while writing params values to a file.")
+			return "", err
+		}
+
+	}
+
+	if certsCM.Name != "" {
+
+		certsDir := filepath.Join(swaggerDirectory, filepath.FromSlash(Deployment+"/"+Certificates))
+		err = utils.CreateDirIfNotExist(certsDir)
+		if err != nil {
+			return "", err
+		}
+
+		certsFileNames, err := maps.ManyKeys(certsCM.Data)
+		if err != nil {
+			logImport.Error(err, "Error in the certs configmap ", "certs cm", certsCM.Data)
+			return "", err
+		}
+
+		for i := 0; i < len(certsFileNames); i++ {
```

Comment:
Is it possible to use for loop directly as follows? So we may not this `maps.ManyKeys` function
```suggestion
		for fileName, data := range controlConf.Data {
```